redfish-exporter/README.md

```diff
@@ -176,15 +106,25 @@ Note that SSL can be enabled using the provided dev certificate and key and `--s
     ./amd-redfish-exporter --enable-slurm
     ```
 
-### Integration Testing
+- The exporter will start subscribing to events from the configured Redfish servers and expose metrics on the specified metrics port.
```

Comment:
They are listed under slurm integration. Could you please check if it's the correct placement?

Reply:
I think there were some formatting issues introduced when I resolved a merge conflict. This is sorted out.